pages/index.js

```diff
@@ -34,9 +25,18 @@ export default function HomePage() {
       ...initialTrips,
     ];
     setTrips(updatedTrips);
+    console.log(updatedTrips);
     event.target.reset();
   }
 
+  function formatDate(date) {
+    const year = date.value.substring(2, 4);
+    const month = date.value.substring(5, 7);
+    const day = date.value.substring(8);
+    const formattedDate = `${day}/${month}/${year}`;
+    return formattedDate;
+  }
```

Comment:
please move this function into a folder called `lib/utils.js` and import it here again (only for best practice reasons). You don't need to repost the PR though.